compiler/testData/codegen/boxInline/nonLocalReturns/fromArrayGenerator.kt

```diff
@@ -0,0 +1,5 @@
+// FILE: 1.kt
+inline fun runReturning(f: () -> Nothing): Nothing = f()
+
+// FILE: 2.kt
+fun box() = Array<String>(1) { runReturning { return@Array "OK" } }[0]
```

Comment:
@pyos Could you also add test with several nested inline calls?
e.g.:
` fun box() = Array<String>(1) { run { runReturning { return@Array "OK" } }}[0] `

and nested calls with capturing
` fun box() { 
val result = "OK"
Array<String>(1) { run { run { runReturning { return@Array result } }}}[0] 
}`

Reply:
Done